workshops/PREFACE.md

```diff
@@ -0,0 +1,27 @@
+# Preface
+
+_Adapted from [here](https://medium.freecodecamp.com/things-i-wish-someone-had-told-me-when-i-was-learning-how-to-code-565fc9dcb329)_
+
+---
+
+If you're reading this, that means that you've decided to embark on the journey of learning to code. Congratulations and welcome, we're excited to have you.
+
+## Coding is Not Special
+
+First and foremost, coding is a skill like any other. Like learning a language, there's grammar and vocabulary to acquire. Like math, there are processes to work through specific types of problems. Like all kinds of craftsmanship and art-making, there are techniques and tools and best practices that people have developed over time, specialized to different tasks, that you're free to use or modify or discard.
```

Comment:
Discard. Into the wastebin. Because some opinions can be garbage.

On Mon, Apr 18, 2016 at 10:15 PM, JESSICA <notifications@github.com> wrote:

> In workshops/PREFACE.md
> <https://github.com/hackclub/hackclub/pull/790#discussion_r60174018>:
>
> > @@ -0,0 +1,27 @@
> > +# Preface
> > +
> > +_Adapted from [here](https://medium.freecodecamp.com/things-i-wish-someone-had-told-me-when-i-was-learning-how-to-code-565fc9dcb329)_
> > +
> > +---
> > +
> > +If you're reading this, that means that you've decided to embark on the journey of learning to code. Congratulations and welcome, we're excited to have you.
> > +
> > +## Coding is Not Special
> > +
> > +First and foremost, coding is a skill like any other. Like learning a language, there's grammar and vocabulary to acquire. Like math, there are processes to work through specific types of problems. Like all kinds of craftsmanship and art-making, there are techniques and tools and best practices that people have developed over time, specialized to different tasks, that you're free to use or modify or discard.
>
> do you mean discard or disregard
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/790/files/b7f32a7938869d17222be04b32487f07d4935bfe#r60174018>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>